aws/src/main/java/org/apache/iceberg/aws/s3/S3FileIOProperties.java

```diff
@@ -98,6 +99,11 @@ public class S3FileIOProperties implements Serializable {
 
   public static final boolean S3_CRT_ENABLED_DEFAULT = true;
 
+  /** This property is used to specify the max concurrency for CRT Async clients. */
+  public static final String S3_CRT_MAX_CONCURRENCY = "s3.crt.max-concurrency";
+
+  public static final int S3_CRT_MAX_CONCURRENCY_DEFAULT = 500;
```

Comment:
can we add some explanation for why 500? Maybe linking to perf results in the accelerator repo README?